src/containers/Display/components/ConfirmModal/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { MouseEventHandler } from "react";
```

Comment:
This modal can be reuse so we should move it into component and turn the message, onClick into props for generic use case